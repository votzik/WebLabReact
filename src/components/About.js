import React, { Component } from "react";
import "./styles/main.css"
import Card from "./Card";
import Footer from "./Footer"

import photo1 from "./assets/puh.jpg"
import photo2 from "./assets/buff.jpg"
import photo3 from "./assets/wat.jpg"
import photo4 from "./assets/yura.jpg"
import photo5 from "./assets/me.jpg"

export default class About extends Component {
    render() {
        return (
            <div>
                <div className="text-mine ml-3 mb-3" >
                    <h1>Our team</h1>
                    <div className="subtext">
                        Here you can look at our immpressive team and it`s members
                    </div>
                </div>
                <ul className="cards-mine">
                    <li>
                        <Card image={photo1}
                            title="Pushok"
                            description="Дизайнер"
                            noButton={true}></Card>

                    </li>
                    <li>
                        <Card image={photo2}
                            title="Buffdan"
                            description="Пише джанго на джанго для джанго"
                            noButton={true}></Card>
                    </li>
                    <li>
                        <Card image={photo3}
                            title="Reitar"
                            description="Дизайнер дизайнера"
                            noButton={true}></Card>
                    </li>

                    <li>
                        <Card image={photo4}
                            title="Yurii"
                            description="він спить не будіть його"
                            noButton={true}></Card>
                    </li>
                    <li>
                        <Card image={photo5}
                            title="Ігор"
                            description="Це Ігор"
                            noButton={true}></Card>
                    </li>
                </ul>
               
               <Footer></Footer>

            </div>
        )
    }
}