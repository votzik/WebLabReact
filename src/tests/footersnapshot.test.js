import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


import About from "../components/About";

configure({ adapter: new Adapter() });
describe('Login', () => {
  it('should render correctly', () => {
    const output = shallow(
      <About />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});