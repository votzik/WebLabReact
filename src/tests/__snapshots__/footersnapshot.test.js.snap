// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<div>
  <div
    className="text-mine ml-3 mb-3"
  >
    <h1>
      Our team
    </h1>
    <div
      className="subtext"
    >
      Here you can look at our immpressive team and it\`s members
    </div>
  </div>
  <ul
    className="cards-mine"
  >
    <li>
      <Card
        description="Дизайнер"
        image="puh.jpg"
        noButton={true}
        title="Pushok"
      />
    </li>
    <li>
      <Card
        description="Пише джанго на джанго для джанго"
        image="buff.jpg"
        noButton={true}
        title="Buffdan"
      />
    </li>
    <li>
      <Card
        description="Дизайнер дизайнера"
        image="wat.jpg"
        noButton={true}
        title="Reitar"
      />
    </li>
    <li>
      <Card
        description="він спить не будіть його"
        image="yura.jpg"
        noButton={true}
        title="Yurii"
      />
    </li>
    <li>
      <Card
        description="Це Ігор"
        image="me.jpg"
        noButton={true}
        title="Ігор"
      />
    </li>
  </ul>
  <Footer />
</div>
`;
